.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,15 @@
+name: AF Auth Layer
```

Comment:
This should have been a separate PR @sudheshna-donthineni . Would have quickly gotten merged by itself.

Reply:
Sorry!